test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -3009,6 +3009,197 @@ public async Task TestPackageManager_PreviewProjectsUninstallPackageAsync_Uninst
             }
         }
 
+        [Fact]
+        public async Task GetTransitivePackagesAsync_WithAssetsFile_ReturnsTransitivePackageReferences()
```

Comment:
naming: 

All tests start with:
`GetTransitivePackagesAsync_WithAssetsFile`. 

I imagine there are different conditions, so the WithAssetsFile part can be replaced with some more descriptive to clarify the conditions.

Reply:
Renamed the tests to be more descriptive.